### DISCUSSION TITLE
How can declare typescript for where cloud?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [prisma/prisma#8930] [Q&A] [answered]
Hi, I have a function like this
```

const getItem= (whereObj /* What typescript can I declare here ?*/)=>{
return await prisma.items.findMany({where:whereObj})
}
```
I found `ItemWhereInput` in prisma type file but I don't know how can import it to use

**Comment by IRediTOTO** [comment]
Look like i found it `Prisma.ProductsFindManyArgs['where']`

**Comment by ryands17** [comment, marked as answer]
@IRediTOTO 👋 
The better way to do this is directly using `ItemWhereInput` like this:

```ts
import { Prisma } from '@prisma/client';

const getItem: Prisma.ItemWhereInput = {};
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
